docs/user_guide/HOWTO/Kerberos.rst

```diff
@@ -60,7 +69,7 @@ Run the following command, as root, to create your principal:
 
   # /usr/sbin/kadmin.local -r YOUR.DOMAIN -q "addprinc <username>/admin"
```

Comment:
kadmin.local is provided by krb5-server, which is not installed by default. Might want to notify them.